Cloud Application Manager/Core Concepts/context-switcher.md

```diff
@@ -0,0 +1,119 @@
+
+![Context Switcher - Search Bar](../../images/cloud-application-manager/context-switcher-search-bar.png)
+
+#### Selection menu
+
+There are several selectable options here depending on user's permissions (organization administrators can see Organizations and Cost Centers options while cost center administrators could only see their associated Cost Centers), and apart from the different kinds of scopes (Organizations, Cost Centers and Workspaces), there is also a Favorites option, which allows to store together those scopes marked as favorites by the user. 
+
+![Context Switcher - Selection Menu](../../images/cloud-application-manager/context-switcher-selection-menu.png)
+
+#### Main view
+
+When an option is selected, main view displays all available scopes of selected kind for a given user, and depending on that kind of scope some differences arise:
+
+***Organizations view***: displays all the organizations the user has access to, and allows him or her to switch between them.
+
+![Context Switcher - Organizations View](../../images/cloud-application-manager/context-switcher-org-view.png)
+
+Every organization, as seen in the previous image, has several elements: name and icon of the organization, a star to mark it (or unmark if already marked) as a favorite, and a pencil (when hovered) to edit its details.
+
+![Context Switcher - Organizations Detail](../../images/cloud-application-manager/context-switcher-org-detail.png)
+
+***Cost Centers view***: this view introduces a left side panel where all the organizations the user has access to are listed, so it can be used as a filter. By default, the *All* option is selected, so every cost center depending on that organizations will be displayed in the main view. Selecting another organization, therefore, will show only its associated cost centers.
+
+![Context Switcher - Cost Centers View](../../images/cloud-application-manager/context-switcher-cc-view.png)
+
+As happened with organizations, a cost center will not only display its name and icon, the favorite star and the editing pencil button (when hovered), but also the icon of the organization it belongs to (bottom left) and the number of members it has (bottom right).
+
+![Context Switcher - Cost Centers Detail](../../images/cloud-application-manager/context-switcher-cc-detail.png)
+
+***Workspaces view***: the most complete view corresponds to workspaces’ because it includes, apart from an organizations panel, a cost centers panel next to it with the same functionality explained for the last section: to act like a filter. This way only associated workspaces of a given cost center will appear.
```

Comment:
```suggestion
##### Workspaces view
The most complete view corresponds to *workspaces* because it includes, apart from an organizations panel, a cost centers panel next to it with the same functionality explained for the last section: to act as a filter. This way only associated workspaces of a given cost center will appear.
```